lib/components/Slider.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import classnames from "classnames";
+import { Slider as BPSlider } from "@blueprintjs/core";
+
+import Label from "./Label";
+
+const Slider = ({
+  label,
+  labelProps,
+  required,
+  helpText,
+  min,
+  max,
+  value,
+  stepSize,
+  labelStepSize,
+  className,
+  onChange}) => {
```

Comment:
All these props except classname can be passed to slider as otherProps.

Reply:
Done